package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "graasp-subscriptions",
```

Comment:
The new convention for plugins is going to be `graasp-plugin-<name>`. Let's follow it with this one. You will also need to change the name of the repo.